src/test/java/com/gooddata/warehouse/WarehouseServiceAT.java

```diff
@@ -85,6 +94,20 @@ public void shouldListUsers() throws Exception {
         assertThat(users.getNextPage(), is(nullValue()));
     }
 
+    @Test(groups = { "warehouse", "isolated_domain" }, dependsOnMethods = "shouldListUsers")
+    public void shouldAddUserToWarehouse() {
+        account = gd.getAccountService().createAccount(new Account(LOGIN, "nnPvcGXU7f", "FistName", "LastName"), getProperty("domain"));
```

Comment:
moreover non working create account call will render this test as failed instead of skipped, which is misleading.

AccountService is not object under test, if you need an user as a fixture, then create it in a beforeclass method, not in the test method